src/main/kotlin/mate/academy/HelloWorldAnonymousClasses.kt

```diff
@@ -8,6 +8,39 @@ class HelloWorldAnonymousClasses {
     }
 
     fun sayHello(names: List<String>): List<String> {
-        // TODO: implement
+        val englishGreeting = object: HelloWorldGreeting {
+            override fun greet(): String = "Hello world"
```

Comment:
It is good practice to move strings to resources or separate variables for hardcode avoiding